plugins/speculation-rules/tests/test-speculation-rules-uninstall.php

```diff
@@ -24,7 +24,7 @@ public static function set_up_before_class(): void {
 	 * Load uninstall.php.
 	 */
 	private function require_uninstall(): void {
-		require __DIR__ . '/../../../plugins/speculation-rules/uninstall.php';
+		require TESTS_REPO_ROOT_DIR . '/plugins/speculation-rules/uninstall.php';
```

Comment:
Why not use `TESTS_PLUGIN_DIR`? Seems more suitable for this.
```suggestion
		require TESTS_PLUGIN_DIR . '/uninstall.php';
```